modules/eks_managed_nodes/variables.tf

```diff
@@ -0,0 +1,79 @@
+variable "tags" {
+  description = "Tags for all resources"
+  type        = map(string)
+  default     = {}
+}
+
+############################
+# K8S Cluster Information
+############################
+variable "cluster_name" {
+  description = "EKS Cluster name"
+  type        = string
+}
+
+variable "worker_iam_role_arn" {
+  description = "Worker Nodes IAM Role ARN"
+  type        = string
+}
+
+variable "cluster_security_group_id" {
+  description = "Security Group ID of the master nodes"
+  type        = string
+}
+
+variable "worker_security_group_id" {
+  description = "Security Group ID of the worker nodes"
+  type        = string
+}
+
+variable "cluster_service_ipv4_cidr" {
+  description = "The CIDR block to assign Kubernetes service IP addresses from. If you don't specify a block, Kubernetes assigns addresses from either the 10.100.0.0/16 or 172.20.0.0/16 CIDR blocks"
+  type        = string
+  default     = null
+}
+
+####################################
+# Node Groups
+# Refer to https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/modules/eks-managed-node-group
+# for the parameters supported. See README for more information.
+####################################
+variable "eks_managed_node_groups" {
+  description = "Map of EKS managed node group definitions to create"
+  type        = any
+  default     = {}
+}
+
+variable "eks_managed_node_group_defaults" {
+  description = "Map of EKS managed node group default configurations"
+  type        = any
+  default = {
+    metadata_options = {
+      http_endpoint               = "enabled"
+      http_tokens                 = "required"
+      http_put_response_hop_limit = 1
+      instance_metadata_tags      = "disabled"
+    }
```

Comment:
These kind of "conflict" with `var.force_imdsv2` and `var.force_irsa` right? Maybe we should just omit this from the defaults?

Reply:
I realise that this is also in the `self_managed_node_groups` variable. I guess we might want to remove it from there too.

Reply:
Done.